test/e2e/proxy_test.go

```diff
@@ -99,3 +139,67 @@ func (tc *testContext) waitForValidTrustedCAConfigMap() error {
 	}
 	return nil
 }
+
+// getSystemEnvVar returns the value corresponding to the input proxy ENV var as set in the registry
+func (tc *testContext) getSystemEnvVar(addr, variableName string) (map[string]string, error) {
+	command := fmt.Sprintf("Get-ChildItem -Path Env: | Where-Object -Property Name -eq '%s' | Format-List ",
+		variableName)
+	out, err := tc.runPowerShellSSHJob(fmt.Sprintf("get-%s-system-env-var",
+		strings.ToLower(strings.ReplaceAll(variableName, "_", "-"))), command, addr)
+	if err != nil {
+		return nil, fmt.Errorf("error running SSH job: %w", err)
+	}
+	value := parseWindowsEnvVars(out)
+	return value, nil
+}
+
+// getServiceProxyEnvVars returns a map of all environment variables present in a service's config
+func (tc *testContext) getServiceProxyEnvVars(addr, svcName string) (map[string]string, error) {
```

Comment:
`getServiceProxyEnvVars` and `getSystemEnvVar` are identical except for the `command`. Please modularize with a helper that is called from each of these functions. Also this function should be `getProxyEnvVarsFromService`. 

```golang
func (tc *testContext) getProxyEnvVarsFromService(addr, svcName string) (map[string]string, error) {
        command := fmt.Sprintf("Get-Process %s | ForEach-Object ...")
        envVars, err  := getEnvVar(addr, svcName, command)
```